src/notifications/debug-logger.js

```diff
@@ -62,19 +63,24 @@ export default {
 
         // NOTE: Raw API does not have callsite.
         const hasCallsite = callsite && callsite.renderSync;
```

Comment:
we have multiple callsite existence checks in the code, should we extract them into the `renderCallsiteSync` method?

Reply:
hm, ok